src/shared/localShadowRepo.ts

```diff
@@ -256,16 +256,13 @@ export class ShadowRepo {
       deployedFiles: deployedFiles.length,
       deletedFiles: deletedFiles.length,
     });
-    // these are stored in posix/style/path format.  We have to convert inbound stuff from windows
-    if (this.isWindows) {
-      this.logger.trace('start: transforming windows paths to posix');
-      deployedFiles = deployedFiles.map(normalize).map(ensurePosix);
```

Comment:
NUTs will determine if anything on windows was relying on the param mutation that we now get warnings about